x-pack/plugins/streams_app/public/routes/config.tsx

```diff
@@ -52,6 +60,15 @@ const streamsAppRoutes = {
               }),
             }),
           },
+          '/{key}/{tab}/{subtab}': {
```

Comment:
should this be `/stream/management/{subtab}`? Not entirely sure it's possible.

Reply:
Good idea, changed!